src/Config.php

```diff
@@ -1658,4 +1644,62 @@ public function printConfigData($data)
     }//end printConfigData()
 
 
+    /**
+     * Assert that all supplied sniff codes have the correct number of parts
```

Comment:
```suggestion
     * Verify that all supplied sniff codes have the correct number of parts.
```

Reason: `assert` is a PHP language construct with its own connotations and is not used here.